### PR TITLE
Adjusts vatborn ages

### PR DESCRIPTION
**Pull request by Schnayy**
Adjusting the min/max age to vatborn to be accurate to [the wiki](https://wiki.ss13polaris.com/index.php?title=Vatborn#.22Natural.22_Vatborn). Lowered the min significantly, lowered the max a wee bit. Added to the blurb a bit about how the aging works to hopefully avoid anyone trying to play as an actual child.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
This feels pretty weird considering vatgrown are more or less identical to baseline humans.

**Comment by elgeonmb** [comment]
alas, 'tis the lore as is written

**Comment by Cerebulon** [comment]
I did suggest implementing minor mechanical changes alongside this besides the existing appendix thing (though mostly for the expedited you want to make a subtype) and I'd bump max age down a little more to like 90 (further for exped. But thats later i guess) but otherwise lorewise is fine.

File: code/modules/mob/living/carbon/human/species/station/human_subspecies.dm [hunk 1]
```diff
@@ -37,9 +37,14 @@
 	name_plural = "Vatborn"
 	blurb = "With cloning on the forefront of human scientific advancement, cheap mass production \
 	of bodies is a very real and rather ethically grey industry. Vat-grown or Vatborn humans tend to be \
-	paler than baseline, with no appendix and fewer inherited genetic disabilities, but a more aggressive metabolism."
+	paler than baseline, with no appendix and fewer inherited genetic disabilities, but a more aggressive metabolism. \
+	Most vatborn are engineered to experience rapid maturation, reaching approximately sixteen years of growth in only five \
```
Comment (Atermonera) [inline code/modules/mob/living/carbon/human/species/station/human_subspecies.dm:41]: [The wiki](https://wiki.ss13polaris.com/index.php?title=Vatborn#.22Natural.22_Vatborn) seems to contradict this claim of "Most".
> The most common form of Vatborn are grown only as infants and "born" in a similar state to standard human babies; this allows the individuals to 'naturally' learn basic human abilities such as body coordination and speech skills as any other child as the brain develops.

> Fully-grown "Expedited" or "Adult" Vatborn are produced in a process typically reserved for cloning. [...] Illegal clone farm operations in violation of the Five Points have been recorded

Comment (Schnayy) [reply code/modules/mob/living/carbon/human/species/station/human_subspecies.dm:41]: > With the exception of "designer babies" ordered by wealthy parents seeking an artificial surrogate, most are engineered to undergo rapid maturation, reaching the equivalent of sixteen years of growth in just five.

**Comment by Atermonera** [comment]
Marking for staff input: This lore is news to me. I've always operated under the assumption that vatborn are baseline humans, but spit-shined and grown in a vat. And I think this lore will be news for other people too. Want to raise awareness because I'm personally dubious whether this follows the Five Points and should be common or uncommon.

**Comment by Schnayy** [comment]
If it's a five points violation to rapid-grow humans, then we should disable cloning.

**Comment by Atermonera** [comment]
Edit: Ater was operating under bad assumptions.

**Review by Atermonera — APPROVED** [review]

**Pull request merged**